File: Modul6/C/PRAK601-2310817310012-MUHAMMAD_AZWIN_HAKIM.c
#include <stdio.h>

int main() {
    int matrix[10][10], baris, kolom;

    scanf("%d %d", &baris, &kolom);

    for (int i = 0; i < baris; i++){
        for (int j = 0; j < kolom; j++){
            scanf("%d", &matrix[i][j]);
        }
    }printf("\n");
    
    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++){
            printf("%d ", matrix[i][j]);
        }
        printf("\n");
    }

    return 0;
}

File: Modul6/Python/PRAK601-2310817310012-MUHAMMAD_AZWIN_HAKIM.py
def buat_matriks(baris, kolom, nilai):
    matriks = [nilai[i:i+kolom] for i in range(0, len(nilai), kolom)]
    return matriks

baris, kolom = map(int, input().split())
nilai = list(map(int, input().split())) 
print()
for baris_matriks in buat_matriks(baris, kolom, nilai):
    print(*baris_matriks)
